src/table/classes.scss

```diff
@@ -28,6 +28,14 @@
 
 .iui-paginator {
   @include iui-paginator;
+
+  .iui-paginator-ellipsis {
+    @include iui-paginator-ellipsis;
+  }
+
+  .iui-paginator-pages-group {
```

Comment:
> iui-paginator-pages-group still needs to go under paginator class, because of button-group rules specificity

Which rules were causing problems? Button group was freshly refactored so I don't like that this is happening

Reply:
For example this one
![image](https://user-images.githubusercontent.com/81580355/147933363-b68ac78e-a749-4d86-8770-0b8c88076175.png)

I was not aware someone refactored button group.  I thought only Button was taken care of..?

Reply:
I guess it was only partly refactored (in the Button PR).

For this issue, you could `@include iui-button-group` and target the same selector (`> *:not(:last-child)`). It will lower the specificity for the rest of the rule

Reply:
Done something